modules/ROOT/pages/cors-policy.adoc

```diff
@@ -60,25 +77,47 @@ The public resource configuration looks like this:
 </http:listener-interceptors>
 ----
 
-If you cannot implement CORS for your API, another possible solution to an unreachable API is to disable the same-origin restrictions in your browser.
+== Disabling Same-Origin Restrictions in Your Browser
+
+If you cannot implement CORS for your API, you can disable the same-origin restrictions in your browser to call an inaccessible API. Each browser handles these restrictions in a unique way. You can research the tips, tricks, and plugins for your browser from the internet.
+
+Before you modify your browser settings, make sure you understand and assess its potential security implications. You must use these browser settings only for testing in your own web pages. Otherwise, the browser can become vulnerable to malicious scripts and other potential threats.
+
+=== Disabling Same-Origin Restrictions on Google Chrome for Mac OS X
+
+To disable same-origin restrictions on Google Chrome for Mac OS X:
+
+* Open a new Terminal window and paste the following line: `open -a Google\ Chrome --args --disable-web-security`.
+* Press *Enter*.
+
+=== Disabling Same-Origin Restrictions on Google Chrome for Windows
 
-== Disable Same-Origin Restrictions in your Browser
+To disable same-origin restrictions on Google Chrome for Windows:
 
-Each browser handles these restrictions in a unique way and tips, tricks, and plugins are available on the internet.
+* Open a command prompt window.
```

Comment:
Is this the same as Terminal above, or are they named differently in Mac and Windows?

Reply:
Steps below:

Reply:
Done! Thanks!